src/control/zoom/ZoomControl.h

```diff
@@ -236,4 +236,7 @@ class ZoomControl: public DocumentListener {
      * depends on dpi (REAL_PERCENTAGE_VALUE * zoom100Value)
      */
     double zoomMin = DEFAULT_ZOOM_MIN;
+
+    size_t current_page = static_cast<size_t>(-1);
+    size_t last_page = static_cast<size_t>(-1);
```

Comment:
Could we use [`npos`](https://github.com/xournalpp/xournalpp/issues/2711) here? 
It looks like [it's equivalent to static_cast<size_t>(-1)](https://www.cplusplus.com/reference/string/string/npos/)...

Reply:
`npos` is a string constant and should only be used parsing strings. We could use `std::numeric_limits<size_t>::max();`
But mostly this is overkill and `static_cast<size_t>(-1)` or `size_t(-1)` will to the same job without reducing readability.